ADAL/src/ADAL_Internal.h

```diff
@@ -54,10 +54,12 @@
 typedef void(^ADAuthenticationCallback)(ADAuthenticationResult* result);
 
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IOS
 //iOS:
 #   include <UIKit/UIKit.h>
 typedef UIWebView WebViewType;
+#elif TARGET_OS_WATCH
+typedef NSString WebViewType; // Should never be called
```

Comment:
Changed it to a more generic NSObject. Since id is a generic pointer type, if we use id here, we will have compilation issue when declare WebViewType properties in other source code like "@property (weak) WebViewType* webView;"